driver-scala/src/main/scala/org/mongodb/scala/model/Filters.scala

```diff
@@ -690,6 +690,14 @@ object Filters {
     JFilters.nearSphere(fieldName, x, y, maxDistance.asJava, minDistance.asJava)
   }
 
+  /**
+   * Creates a filter that will match all documents.
```

Comment:
Nit: Keep this the same as the Javadoc: ` Creates an empty filter that will match all documents.`

Reply:
done